scripts/test.js

```diff
@@ -85,19 +85,24 @@ const tasks = {
     option: '--runInBand',
     extraParam: '--runInBand',
   }),
+  update: createOption({
```

Comment:
That's good that we're bringing this option to master, but let's backport the documentation as well then